deploy/helm/scf/assets/operations/instance_groups/api.yaml

```diff
@@ -46,11 +56,21 @@
   path: /instance_groups/name=api/jobs/name=file_server/properties/set_kernel_parameters?
   value: false
 
+# We don't have a /var/vcap/job/*/packages directory, so we point to all the packages
+- type: replace
+  path: /instance_groups/name=api/jobs/name=file_server/properties/diego?/file_server?/static_directory?
```

Comment:
`?` is transient to the right, meaning we only need it on `diego` (and we should only set on diego if it doesn't exist).